engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
@Horneth Yes, with the change for your comment (the first of this thread i.e.), WA will need the MaterializeActor